content/en/blog/2024/ambient-reaches-beta/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Istio’s new sidecar-less ‘Ambient Mode’ reaches Beta in version 1.22"
+description: The latest release from Istio brings service mesh Layer 4 & 7 features to production readiness without sidecars.
+publishdate: 2024-05-14
+attribution: "Lin Sun (Solo.io), for the Istio Steering and Technical Oversight Committees"
+keywords: [Istio Day,Istio,conference,KubeCon,CloudNativeCon]
+---
+
+Istio ambient service mesh was [announced in September 2022](/blog/2022/introducing-ambient-mesh/) as an experimental branch
+that introduced a new data plane mode in Istio that did not require sidecars, offering a simplified operational experience of
+Istio. After 20 months of hard work and collaboration within the Istio community, with contributions from  Solo.io, Google,
+Microsoft, Intel, Aviatrix, Huawei, IBM, Red Hat, and others, we are excited to announce that ambient mode has reached Beta in
+version 1.22! The beta release of 1.22 indicates the features of ambient mode are now stable and ready for production workloads
+with appropriate cautions.  This is a huge milestone for Istio, bringing both Layer 4 and Layer 7 mesh features to production
```

Comment:
```suggestion
with appropriate precautions.  This is a huge milestone for Istio, bringing both Layer 4 and Layer 7 mesh features to production
```

Reply:
Feels it is cautions, will wait till others to weigh in.

Reply:
`precautions` makes sense if we are referring to steps that a user will take to ensure we are carefully running production workloads.

`caution` a term meaning to avoid danger. In this case we are telling the user to be cautious when running ambient so I don't think `cautions` makes sense but to act with `caution` does. Wdyt?
```suggestion
with caution.  This is a huge milestone for Istio, bringing both Layer 4 and Layer 7 mesh features to production
```

Reply:
The way I read it was steps to be taken before doing, which is a precaution. IE, in this case, we're still saying there are some elements that are alpha, which they should check beforehand.

Reply:
Yeah that makes sense just wanted to point out that I don't think `cautions` is right if we went that route. But yeah, `precautions` seems to be the word we're looking for here.

Reply:
Thanks! updated to precautions!